documentation/src/main/asciidoc/mapping.asciidoc

```diff
@@ -2935,6 +2952,16 @@ public class Departments {
 
 ==== Mapping dynamic boost
 
+[WARNING]
+====
+Index-time boosting is deprecated: it will be removed starting from Lucene 7.
+
+To implement static boosting, you can use query-time boosting instead: see <<search-query>>.
```

Comment:
Will replace with the same warning as in the "Dynamic index time boosting" section.